Gemfile

```diff
@@ -1,12 +1,20 @@
+# frozen_string_literal: true
+
 source "https://rubygems.org"
-# If you bump the Ruby version, make sure to update the Vagrantfile appropriately
+
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:gitlab) { |repo| "https://gitlab.com/#{repo}.git" }
+
+# Update Vagrantfile and CircleCI config if Ruby version is bumped
 ruby "2.5.5"
-gem "rails", "4.2.11"
+gem "rack", "2.0.8"
+gem "rails", "5.2.4"
 
 gem "active_model_serializers", "~> 0.9.3"
 gem "bcrypt", "~> 3.1.7"
+gem "bootsnap"
 gem "jquery-rails"
-gem "pg"
+gem "pg", "~> 1.0"
```

Comment:
pg is pinned to `~> 1.0` here to fix / minimize breaking interface changes. Same with the dependencies below.